src/titiler/core/titiler/core/templates/wmts.xml

```diff
@@ -51,13 +51,13 @@
             </TileMatrixSetLink>
             <ResourceURL format="{{ media_type }}" resourceType="tile" template="{{ layer.tiles_url }}?{{ layer.query_string | escape }}" />
         </Layer>
-    {% endfor %}
+    {%- endfor %}
```

Comment:
why do we need `-` ?

Reply:
it removes extra white space that's being inserted https://jinja.palletsprojects.com/en/stable/templates/#whitespace-control